test/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ message(STATUS "Adding Tests")
 
 set(CLEsperantoTests
   itkCLEImageTest.cxx
+  itkCLEAddImageTest.cxx
```

Comment:
In addition to adding the source file reference here, another test entry needs to be added with `itk_add_test` (see example on line 12 below).